CMSgov/cms-carts-seds
# MDCT-105 - State View

# Description

Allow the state user to view all reports for their state.
Additionally provides some functionality for the other user types to view all reports, and retrieve some dummy data from the /state endpoint so those views don't crash and burn.

Sets up seed StateStatuses for the local environment for Alabama. This is easy enough to iterate on and expand if other entries are needed.

## How to test

* Running locally, and logging in as StateUser2 should allow you to see Statuses for Alabama, and only Alabama. 
* The links for View/Edit should exist based on the Report Status.
* A modified date has been added for each entry for the stateUser

## Dependencies 

The .env_example file in the root level has a few new props that you will want to copy into your .env to be able to run the app.

# Get it done

Look through, Test and Approve

## Code authors checklist

## Reviewers Checklist (Two different people)

## Assignee
